gdk-pixbuf/Gir.toml

```diff
@@ -39,6 +39,8 @@ status = "generate"
     name = "get_pixels"
     #manual array without length
     manual = true
+    # docs were manually written to include a section on safety
+    generate_doc = false
```

Comment:
You'll also need to update the `gir` submodule and re-run `generator.py`

Reply:
Done that in https://github.com/gtk-rs/gtk-rs-core/pull/637 so this can just be merged once that is in